Source/DataTypes/SvgViewBox.cs

```diff
@@ -242,8 +242,8 @@ public override object ConvertTo(ITypeDescriptorContext context, CultureInfo cul
                 var viewBox = (SvgViewBox)value;
 
                 return string.Format("{0}, {1}, {2}, {3}",
-                    viewBox.MinX.ToString(CultureInfo.InvariantCulture), viewBox.MinY.ToString(CultureInfo.InvariantCulture),
-                    viewBox.Width.ToString(CultureInfo.InvariantCulture), viewBox.Height.ToString(CultureInfo.InvariantCulture));
+                    viewBox.MinX.ToString(CultureInfo.InvariantCulture), viewBox.MinY.ToSvgString(),
+                    viewBox.Width.ToString(CultureInfo.InvariantCulture), viewBox.Height.ToSvgString());

```

Comment:
Have you checked if this impacts the performance?

Reply:
No, will create benchmark.

Reply:
Performance is the same.

BenchmarkDotNet=v0.12.1, OS=Windows 10.0.19042
Intel Core i7-4790K CPU 4.00GHz (Haswell), 1 CPU, 8 logical and 4 physical cores
.NET Core SDK=5.0.101
  [Host]     : .NET Core 3.1.10 (CoreCLR 4.700.20.51601, CoreFX 4.700.20.51901), X64 RyuJIT
  Job-AOWSNB : .NET Core 3.1.10 (CoreCLR 4.700.20.51601, CoreFX 4.700.20.51901), X64 RyuJIT

Runtime=.NET Core 3.1

|            Method |     Mean |   Error |  StdDev |        Op/s | Ratio | Rank |  Gen 0 | Gen 1 | Gen 2 | Allocated |
|------------------ |---------:|--------:|--------:|------------:|------:|-----:|-------:|------:|------:|----------:|
|    Float_ToString | 638.4 ns | 1.68 ns | 1.49 ns | 1,566,483.8 |  1.00 |    1 | 0.0381 |     - |     - |     160 B |
| Float_ToSvgString | 640.9 ns | 2.20 ns | 1.95 ns | 1,560,194.7 |  1.00 |    1 | 0.0381 |     - |     - |     160 B |